pcstac/tests/resources/test_item.py

```diff
@@ -153,6 +162,30 @@ async def test_item_search_temporal_window_post(app_client):
     assert resp_json["features"][0]["id"] == first_item["id"]
 
 
+@pytest.mark.asyncio
+async def test_item_search_temporal_window_post_date_only(app_client):
+    """Test POST search with spatio-temporal query (core)"""
+    items_resp = await app_client.get("/collections/naip/items")
+    assert items_resp.status_code == 200
+
+    first_item = items_resp.json()["features"][0]
+    item_date = UtcDatetimeAdapter.validate_strings(
+        first_item["properties"]["datetime"]
+    )
+    item_date_before = item_date - timedelta(days=1)
+    item_date_after = item_date + timedelta(days=1)
+
+    # NOTE: YYYY-MM-DD format is not supported anymore
+    params = {
+        "collections": first_item["collection"],
+        "bbox": ",".join([str(coord) for coord in first_item["bbox"]]),
+        "datetime": f"{item_date_before.strftime('%Y-%m-%d')}/"
+        f"{item_date_after.strftime('%Y-%m-%d')}",
+    }
+    resp = await app_client.post("/search", json=params)
+    assert resp.status_code == 400
```

Comment:
⚠️ in previous version it seems that `YYYY-MM-DD` was supported, this is not supported anymore.

we could adapt the Search model to relaxe the validation to accept the `old` format 

cc @mmcfarland

Reply:
I think that'd be best. Reviewing the example notebooks we render on the site, we do demonstrate using `YYYY-MM-DD`

Reply:
resolved in https://github.com/microsoft/planetary-computer-apis/pull/222/commits/262dc53167abb66469243eb5c9eff68fb994e6d1